golang/1.24/versions.sh

```diff
@@ -0,0 +1,2 @@
+GO_VERSION=1.24.3
+GOLANGCI_LINT_VERSION=2.1.6
```

Comment:
this will break the update logic https://github.com/scylladb/scylla-operator-images/blob/69741464fd17785b3b9a3508270ca0e5b97c06e7/golang/update.sh#L22, I'm not sure GOLANGCI_LINT_VERSION is needed though